js/index.js

```diff
@@ -80,10 +80,31 @@ function addCard() {
 			.reverse()[0]
 	);
 
+	if($("cards").hasChildNodes() && $("cards").firstChild.nodeName == "#text") {
+		$("cards").removeChild($("cards").firstChild)
+	}
 	$("cards").appendChild(clone);
 	impl.genPermalink();
 }
 
+function flipCards() {
+	const cards = $("cards");
+	if (!cards) return;
+	let card_nodes = [];
+	while (cards.hasChildNodes()) {
+		if(cards.firstChild.nodeName == "#text"){
```

Comment:
flip ボタンを押したら全消しになったんですけど、この条件逆じゃないですか？
```suggestion
		if(cards.firstChild.nodeName !== "#text"){
```

Reply:
あ、ホントですね

Reply:
https://github.com/Qithub-BOT/mastogetter/pull/81/commits/abba4549d438b40a91cba00472dfeb12ae1c3319
こちらで修正しました。